### PR TITLE
add support for a phpsu.phar file

### PR DESCRIPTION
**Pull request by Kanti**
## Description and Background

I just stumbled across humbug/box. It is very nice.
related: #102

## Motivation and Context

It would be very nice If phpsu could be used as .phar file.

#### Checklist

- [x] My code follows the PSR coding guideline.
- [ ] I have updated the `documentation` accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added `tests` to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/Helper/ApplicationHelper.php [hunk 1]
```diff
@@ -12,7 +12,19 @@ final class ApplicationHelper
...
+        return $this->getPhpSuVersionFromGlobals() ?? $this->getPhpSuVersionFromVendor() ?? $this->getPhpSuVersionFromGitFolder() ?? 'development';
+    }
+
+    /**
+     * @return string|null
+     */
+    private function getPhpSuVersionFromGlobals()
+    {
+        $pharVersion = '@phpsu_version@';
+        if (strpos($pharVersion, '@') === 0) {
```
Comment (ChrisB9) [inline src/Helper/ApplicationHelper.php]:
```suggestion
        if (\strpos($pharVersion, '@') === 0) {
```

File: src/Helper/ApplicationHelper.php [hunk 2]
```diff
@@ -12,7 +12,19 @@ final class ApplicationHelper
...
-        return $this->getPhpSuVersionFromVendor() ?? $this->getPhpSuVersionFromGitFolder() ?? 'development';
+        return $this->getPhpSuVersionFromGlobals() ?? $this->getPhpSuVersionFromVendor() ?? $this->getPhpSuVersionFromGitFolder() ?? 'development';
+    }
+
+    /**
+     * @return string|null
+     */
+    private function getPhpSuVersionFromGlobals()
+    {
+        $pharVersion = '@phpsu_version@';
```
Comment (ChrisB9) [inline src/Helper/ApplicationHelper.php]: how does this return anything besides null? is box auto-replacing this on build?

Comment (Kanti) [reply src/Helper/ApplicationHelper.php]: yes

Comment (ChrisB9) [reply src/Helper/ApplicationHelper.php]: :+1:

**Pull request merged**